ERCS/erc-7730.md

```diff
@@ -0,0 +1,1033 @@
+
+> *Deployment model*
+> 
+> Making the ERC-7730 available for wallets is a key factor of adoption. We have a few options:
+> - In each dApps github repository or web site: good for autonomy but can be problematic for discoverability by wallets of new ERC-7730 files
+> - Foundation operated repository, like ethereum chainID list: good alternative between decentralization and discoverability.
+> - Ledger repository: as a short term solution, Ledger is providing a central repository (See Ledger github repository)
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Simple example
+
+The following is a simple example of how to clear sign a `transfer` function call on an [ERC-20](./eip-20.md) contract.
+
+```json
+{
+    "$schema": "https://github.com/LedgerHQ/clear-signing-erc7730-registry/blob/master/specs/erc7730-v1.schema.json",
+
+    "context": {
+        "$id": "Tether USD",
+        "contract" : {
+            "abi": "https://api.etherscan.io/api?module=contract&action=getabi&address=0xdac17f958d2ee523a2206206994597c13d831ec7",
+            "deployments": [ 
+                {
+                    "chainId": 1,
+                    "address": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+                },
+                {
+                    "chainId": 137,
+                    "address": "0xc2132D05D31c914a87C6611C10748AEb04B58e8F",
+                },
+                {
+                    "chainId": 42161,
+                    "address": "0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9",
+                }
+            ]
+        }
+    }, 
+
+    "metadata": {
+        "owner": "Tether",
+        "info": {
+            "legalName": "Tether Limited",
+            "url": "https://tether.to/",
+            "lastUpdate": "2017-11-28T12:41:21Z"  
+        }
+    },
+
+    "display": {
+        "formats": {
+            "transfer(address,uint256)": {
+                "intent": "Send",
+                "fields": {
+                    "_to" : {
+                        "label": "To",
+                        "format": "addressOrName"
+                    },
+                    "_value" : {
+                        "label": "Amount",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "@.to"
+                        }
+                    }
+                },
+                "required": ["_to", "_value"]
+            }
+        }
+    }
+}
+```
+
+The `$schema` key refers to the latest version of this specification json schema (currently version 1).
```

Comment:
> (currently version 1)

Once this proposal goes final it can no longer be updated, so when you publish version 2 this text would become misleading. One option is:

```suggestion
The `$schema` key refers to the latest version of this specification json schema (version 1 at the time of writing).
```